CreeDictionary/utils/project_path.py

```diff
@@ -0,0 +1,39 @@
+"""
+This module provides utility that resolves our project paths listed in pyproject.toml
+"""
+import toml
+from typing import Dict, Optional
+from pathlib import Path
+from os import path
+
+FILE_DIR = Path(path.dirname(__file__))
+
+
+def _get_project_root() -> Path:
+    """
+    bubbles up until we see a directory with .git folder
+
+    :return: The root directory of the repository
+    """
+    # Uhh I actually don't know any reliable way to do this
+    # this sounds like a reasonable method. You are Welcome to fix it.
+    current = FILE_DIR
+    while not (current / ".git").is_dir():
+        current = current.parent
+    return current
```

Comment:
This makes the assumption that the code is always in a git repository, which is not always the case. I'm not sure of a fool-proof way of doing this without making some assumption that can easily change :/

I think a better assumption is the `pyproject.toml` since it IS a runtime dependency — `.git` is technically a development thing.

```suggestion
    # Uhh I actually don't know any reliable way to do this
    # this sounds like a reasonable method. You are welcome to fix it.
    current = FILE_DIR
    while not (current / "pyproject.toml").is_file():
        current = current.parent
    return current
```